src/Tizen.Security.TEEC/Tizen.Security.TEEC/Libteec.cs

```diff
@@ -345,7 +346,14 @@ public sealed class Session
         /// </summary>
         ~Session()
         {
-            Close();
+            Dispose();
```

Comment:
Please apply Dispose pattern 

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/dispose-pattern

Reply:
It cause double free, When Dispose was called by user